conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, append to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have a recipes directory containing .bb and .bbappend files, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes*/*/*.bb \
+            ${LAYERDIR}/recipes*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "xt-rpi5"
+BBFILE_PATTERN_xt-rpi5 := "^${LAYERDIR}/"
+BBFILE_PRIORITY_xt-rpi5 = "10"
+
+LAYERSERIES_COMPAT_xt-rpi5 = "kirkstone scarthgap"
```

Comment:
Is this kirkstone compatible? I suspect it is not.

Reply:
In theory - should be compatible. But I didn't check